lib/Settings/MappingProfiles/MappingProfiles.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Pane,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import { SettingsLabel } from '../SettingsLabel';
+import { SearchForm } from '../../SearchForm';
+import { listTemplate } from './listTemplate';
+
+const MappingProfiles = props => {
+  const {
+    columnMapping,
+    columnWidths,
+    visibleColumns,
+    contentData,
+  } = props;
+
+  return (
+    <Pane
+      data-test-mapping-profiles-pane
+      defaultWidth="fill"
+      paneTitle={(
+        <SettingsLabel
+          messageId="stripes-data-transfer-components.mappingProfilesTitle"
+          iconKey="mappingProfiles"
+          app="data-export"
+        />
+      )}
+    >
+      <SearchForm
+        searchTerm=" "
+        searchLabelKey="stripes-data-transfer-components.search"
+      />
+      <MultiColumnList
+        id="mapping-profiles-list"
+        totalCount={contentData.length}
+        contentData={contentData}
+        columnMapping={columnMapping}
+        visibleColumns={visibleColumns}
+        columnWidths={columnWidths}
+        formatter={listTemplate}
+        autosize
+      />
+    </Pane>
+  );
+};
+
+MappingProfiles.propTypes = {
+  visibleColumns: PropTypes.arrayOf(PropTypes.string),
+  columnMapping: PropTypes.object,
+  columnWidths: PropTypes.object,
+  contentData: PropTypes.arrayOf(PropTypes.object),
+};
+
+MappingProfiles.defaultProps = {
+  visibleColumns: [
+    'name',
+    'folioRecord',
+    'tags',
```

Comment:
As far as I see data-export is not going to have `tags` column. Instead, it will have `format`. Please take a look at [mockups](https://drive.google.com/drive/folders/1qUP38ooZ_ZqW9RNH-53j65sZv6jSiZhx).